src/util/containers/ContainerManager.ts

```diff
@@ -0,0 +1,27 @@
+import type { ResourceIdentifier } from '../../ldp/representation/ResourceIdentifier';
+
+/**
+ * Handles everything related to container identifiers and how they relate to other identifiers.
+ */
+export interface ContainerManager {
+  /**
+   * Verifies if this identifier could be valid.
+   * This does not check if this identifier actually exists,
+   * but checks if the identifier is something that could be a valid identifier.
+   */
+  isValidIdentifier: (identifier: ResourceIdentifier) => boolean;
+
+  /**
+   * Generates the identifier of the container this resource would be a member of.
+   * This does not check if that identifier actually exists, this is simply string manipulation.
```

Comment:
```suggestion
   * This does not check if that identifier actually exists, this is simply string manipulation.
```

Or an oracle, we don't know that 😄